### DISCUSSION TITLE
running madymo-openradioss coupling

### DISCUSSION THREAD

**Discussion by batkillerz** [OpenRadioss/OpenRadioss#1515] [Q&A] [answered]
Hi, is it possible to run a madymo-openradioss coupling?

**Comment by OlivierWienholtz** [comment, marked as answer]
Hi BatKillerz

No, 
The coupling with Madymo is not available on OpenRadioss.

Best Regards
Olivier

**Comment by batkillerz** [comment]
Thanks!
